app/controllers/organization/organizations_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Organization::OrganizationsController < ApplicationController
+  # load_and_authorize_resource
+  before_action :set_organization, only: %w[edit update show]
+
+  def index
+    @organizations = Organization.accessible_by(current_ability, :manage)
+  end
+
+  def new
+    @organization = Organization.new
+    @user = @organization.users.build
+  end
+
+  def create
+    @organization = Organization.new(organization_params)
+    if @organization.save
+      @organization.users.first.add_role :admin
+      redirect_to redirect_path, notice: 'Invitation sent to email.'
+    else
+      render :new
+    end
+  end
+
+  def show; end
+
+  def edit; end
+
+  private
+
+  def redirect_path
+    current_user.present? ? organizations_path : new_user_session_path
+  end
+
+  def set_organization
```

Comment:
Change method name to `organization`.

Reply:
handle with memoization 

`@organization ||= Organization.find_by(id: params[:id])`